backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -292,7 +294,7 @@ public String getOrderingProviderID() {
 
   @JsonProperty("First_test")
   public String getFirstTest() {
-    return boolToYesNoUnk(survey.map(AskOnEntrySurvey::getFirstTest));
+    return TRUE.equals(testEvent.getPatientHasPriorTests()) ? "N" : "UNK";
```

Comment:
**praise:** Good call using `TRUE.equals` here – this means that if we have to resend older test events they'll render `UNK` rather than crashing in serialization